.travis.yml

```diff
@@ -33,9 +33,9 @@ env:
     - STRIPE_MOCK_VERSION=0.40.0
 
 go:
-  - "1.9.x"
-  - "1.10.x"
-  - "1.11.x"
```

Comment:
Probably worth testing against the latest patch release still

Reply:
Oops — nice catch. Fixed!